projects/vdk-core/.gitlab-ci.yml

```diff
@@ -18,6 +18,8 @@ image: "python:3.9"
   stage: build
   before_script:
     - cd projects/vdk-core/
+    - pip install -U pip
+    - pip install ./plugins/vdk-test-utils # TODO: REMOVE THIS AT SOME POINT
```

Comment:
why is that necessary? build.sh does the same `pip install -e plugins/vdk-test-utils`

Reply:
ditched it, ditched the line under it as well as it was a duplicate too